pointcloud/basic_octree.go

```diff
@@ -61,9 +61,9 @@ func (octree *BasicOctree) Size() int {
 	return octree.size
 }
 
-// MaxProb returns the octree's max probability.
-func (octree *BasicOctree) MaxProb() float64 {
-	return octree.node.maxProb
+// MaxVal returns the max value of all children's data for the passed in octree.
```

Comment:
[nit] children's -> descendants'

Reply:
I'd suggest we keep it children as that type of node is usually referenced to as a child or children node when talking about octree/quadtrees. In addition, that is what they are referred to in the octree `node` struct

Reply:
Ok, sounds good